mteb/tasks/Classification/san/SanskritShlokasClassification.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
```

Comment:
Please run `make lint` or `make pr` and push changes

Reply:
Done